docs/user/configuration/server-config.md

```diff
@@ -53,4 +53,61 @@ The `server_tls_config` configures TLS.
 
 # File path to the signing CA certificate, needed if CA is not trusted
 [client_ca_file: <string>]
+
+# Windows certificate filter allows selecting client CA and server certificate from the Windows Certificate store
+[windows_certificate_filter: <windows_certificate_filter_config>]
+```
+
+## windows_certificate_filter_config
+
+The `windows_certificate_filter_config` configures the use of the Windows Certificate store. Setting cert_file, key_file, and client_ca_file are invalid settings when using the windows_certificate_filter.
+
+```yaml
+# Client configuration, optional. If nothing specific will use the default client ca root
+[client: <windows_client_config>]
+  
+# Name of the store to look for the Client Certificate ex My, CA
+[server: <windows_server_config>]
 ```
+
+
+### windows_client_config
+
+```yaml
+# Name of the system store to look for the Client Certificate ex LocalMachine, CurrentUser 
+[system_store: <string>]
+
+# Name of the store to look for the Client Certificate ex My, CA
+[store: <string>]
```

Comment:
Are we sure these aren't required? What will happen if there's no system store / store set?